server/app/views/admin/programs/ProgramBlockAllAspectsView.java

```diff
@@ -243,19 +264,27 @@ private DivTag renderBlockList(
                   "border",
                   "border-white",
                   StyleUtils.hover("border-gray-300"),
-                  selectedClasses)
-              .with(
-                  a().withClasses("flex-grow", "overflow-hidden")
-                      .withHref(editBlockLink)
-                      .with(p(blockName), p(questionCountText).withClasses("text-sm")))
-              .with(moveButtons);
+                  selectedClasses);
 
+        String switchBlockLink =
+          controllers.admin.routes.AdminProgramBlocksController.edit(
```

Comment:
same as my other comment, edit() will throw an exception if loaded on an Active program which I believe this is rendered for.